src/utils/getPercentageOfDaysLeft.ts

```diff
@@ -1,13 +1,8 @@
 import moment from 'moment';
 
-function inputParser(input: string) {
-    const parsedDate = moment(new Date(parseInt(input, 10) * 1000));
-    return parsedDate;
-}
-
 const getPercentageOfDaysLeft = (startedOn: string, endsOn: string): number => {
-    const startDate = inputParser(startedOn);
-    const endDate = inputParser(endsOn);
+    const startDate = moment(new Date(parseInt(startedOn, 10) * 1000));
+    const endDate = moment(new Date(parseInt(endsOn, 10) * 1000));
```

Comment:
Why was inputParser removed here? The function could have been used to achieve the same - you wouldn't have had the need to repeat this: `moment(new Date(parseInt(_, 10) * 1000))`
The only difference is `_`. other than that it's the same code.